### PR TITLE
Update default issues and PR templates

### PR DESCRIPTION
**Pull request by pmahnke**
## Done

- Removed the takeover issue template as we no longer use this
- Updated the default PR template to make it more useful for the common usecase

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- [List additional steps to QA the new features or prove the bug has been resolved]

### CODE REVIEW

**Review by bethcollins92 — APPROVED** [review]
LGTM - I think if you rebase on `main` the test python check should pass

**Comment by petermakowski** [comment]
@pmahnke  Please rebase with `main` after this gets merged https://github.com/canonical-web-and-design/ubuntu.com/pull/10768 and cypress checks will pass.

**Pull request merged**
